plugins/synapticsmst/synapticsmst-device.c

```diff
@@ -377,7 +384,7 @@ synapticsmst_device_enumerate_device (SynapticsMSTDevice *device,
 	/* read board ID */
 	if (!synapticsmst_device_read_board_id (device, connection, byte, error))
 		return FALSE;
-	priv->board_id = (byte[0] << 8) | (byte[1]);
+	priv->board_id = (byte[1] << 8) | (byte[0]);
```

Comment:
I think this is likely going to cause CI to fail since the test suite (with captured files) inverted the order of these bytes.  So might need to flip the bytes filled in line 313 and 314 of this PR.

Reply:
Fixed in latest push.